.github/workflows/go.yml

```diff
@@ -209,6 +212,7 @@ jobs:
       if: ${{ github.base_ref == 'main' }}
       uses: gaurav-nelson/github-action-markdown-link-check@v1
       with:
+        check-modified-files-only: yes
```

Comment:
I think this is a good idea, but how does `check-modified-files-only: yes` behave for the cron job?

Reply:
Removed schedule from go.yml and created a new workflow for cron job.